xbmc/platform/darwin/DarwinUtils.mm

```diff
@@ -407,25 +398,23 @@ enum iosPlatform getIosPlatform()
     {
       rootFolder = "Library/Preferences";
     }
-  }
+  });
   return rootFolder.c_str();
 }
 
 bool CDarwinUtils::IsIosSandboxed(void)
 {
-  static int ret = -1;
-  if (ret == -1)
-  {
+  static int ret = 0;
```

Comment:
With the `dispatch_once` (or `std::call_once`) change this can be changed to bool.